tableauserverclient/server/endpoint/auth_endpoint.py

```diff
@@ -41,6 +41,9 @@ def sign_in(self, auth_req):
 
     def sign_out(self):
         url = "{0}/{1}".format(self.baseurl, 'signout')
+        # If there are no auth tokens you're already signed out. No-op
+        if not self.parent_srv._auth_token:
```

Comment:
When you say "signed in" are you just defining that as having a token? 
The token isn't guaranteed to be valid just because it exists (the crux of this issue) which means you aren't actually 'signed in' -- but if that's ok I can have something like that